### PR TITLE
AuthApp modal: Display an app card when registering for the first time

### PR DESCRIPTION
**Pull request by G-Ray**

### CODE REVIEW

**Review by leventdem — APPROVED** [review]
ok

File: src/modals/AuthApp/AuthApp.jsx [hunk 1]
```diff
@@ -72,6 +72,7 @@ class AuthApp extends React.Component {
     if (appRequest.isConnected === false) {
       return (
         <div>
+          <Card minHeight={64} title={appRequest.name} image={appRequest.imageURL} color='#a3005c' description={appRequest.description} />
           <p className={styles.description}>
           Cette notification apparait car cette application demande un accès à votre stockage Masq.
```
Comment (sujeebant) [inline src/modals/AuthApp/AuthApp.jsx]: Replace by 'Cette application demande l'accès à votre stockage Masq.' ?
'Cette notification apparait car' may not be useful.

File: src/modals/AuthApp/AuthApp.jsx [hunk 2]
```diff
@@ -96,9 +97,10 @@ class AuthApp extends React.Component {
...
   render () {
     const { appRequest } = this.props
+    const { isConnected } = appRequest
 
     return (
-      <Modal width={511} height={400}>
+      <Modal width={511} height={isConnected === false ? 600 : 400}>
         <div className={styles.AuthApp}>
           <p className={styles.title}>Nouvelle requête de connexion de:</p>
           <p className={styles.appTitle}>{appRequest.appId}</p>
```
Comment (sujeebant) [inline src/modals/AuthApp/AuthApp.jsx]: May not be useful as the Card appears underneath ?

**Review by sujeebant — APPROVED** [review]

File: src/modals/AuthApp/AuthApp.jsx (hunk 1 above)
Comment (leventdem) [reply src/modals/AuthApp/AuthApp.jsx]: ok

**Pull request merged**